test/s1/appframework/appframework_test.go

```diff
@@ -475,6 +479,12 @@ var _ = Describe("s1appfw test", func() {
 			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
+			// Upload apps to S3 for Standalone
+			s3TestDir := "s1appfw-" + testenv.RandomDNSName(4)
+			uploadedFiles, err = testenv.UploadFilesToS3(testS3Bucket, s3TestDir, appFileList, downloadDirV1)
```

Comment:
Don't we need to repopulate appFileList here?

Reply:
The app list is not changing across MC and S1 source so we are using the same variable.